AsyncDisplayKit/Private/ASInternalHelpers.h

```diff
@@ -49,6 +49,13 @@ ASDISPLAYNODE_INLINE void ASPerformBlockWithoutAnimation(BOOL withoutAnimation,
   }
 }
 
+ASDISPLAYNODE_INLINE void ASBoundsAndPositionForFrame(CGRect rect, CGPoint origin, CGPoint anchorPoint, CGRect *bounds, CGPoint *position)
+{
+  *bounds      = (CGRect){ origin, rect.size };
+  *position    = CGPointMake(rect.origin.x + rect.size.width * anchorPoint.x,
```

Comment:
Spacing could be improved -- less before the equals, one additional space on the second line of arguments to align the start of them